libs/api/domains/auth/src/lib/resolvers/delegationScope.resolver.ts

```diff
@@ -19,19 +19,11 @@ export class DelegationScopeResolver {
 
   @ResolveField('type')
   resolveType(@Parent() delegationScope: DelegationScopeDTO): string {
-    return delegationScope.scopeName
-      ? ScopeType.ApiScope
-      : delegationScope.identityResourceName
-      ? ScopeType.IdentityResource
-      : ''
+    return delegationScope.scopeName ? ScopeType.ApiScope : ''
```

Comment:
When is this condition false?

Reply:
Never, after I removed IdentityResources so I updated this to return just `ScopeType.ApiScope`